.github/ISSUE_TEMPLATE/Bug_report.md

```diff
@@ -8,10 +8,12 @@ about: Create a report to help us improve
 A clear and concise description of what the bug is.
 
 **To Reproduce**
-Steps to reproduce the behavior:
-1. Go to '...'
-2. Click on '....'
-3. Scroll down to '....'
+A clear and concise description of steps to reproduce the behavior.
+
+For example:
+1. Open Terminal

```

Comment:
Better:
1. Have certain environment
1. Run given code snippet in a certain way
1. See some behavior described